### PR TITLE
Support for GoDaddy Website Builder

### PR DESCRIPTION
**Pull request by asilvas**

### CODE REVIEW

**Comment by rviscomi** [comment]
Thanks @asilvas!

`dps` is a very short fingerprint, especially when it's looked up anywhere in the response headers. So I'm worried about false positives.

Looks like it's being detected on 3764 websites. Out of curiosity, I see on your GH profile that you work for GoDaddy, so you might be able to modify the query to produce a list of detected URLs that you could cross-reference with internal customer data. Also, does 3764 websites out of ~6M seem reasonable?

**Comment by asilvas** [comment]
Hey @rviscomi, good point. Technically the header in question is `server: DPS/1.6.13`, so not ideal based on the current expression. I'll update the expression to `dps/` to avoid potential collisions. I am surprised to see there aren't far more website results though, considering we have hundreds of thousands online.

**Comment by asilvas** [comment]
I verified several results and all were correct. That doesn't explain why the dataset only contains a few thousands of these websites, but at least the query seems to be accurate.

File: ttfb.sql [hunk 1]
```diff
@@ -31,7 +32,7 @@ FROM
   UNNEST(experimental.time_to_first_byte.histogram.bin) AS ttfb
 JOIN
   (SELECT _TABLE_SUFFIX AS client, url, REGEXP_EXTRACT(LOWER(CONCAT(respOtherHeaders, resp_x_powered_by, resp_via, resp_server)),
-      '(seravo|x-kinsta-cache|automattic.com/jobs|x-ah-environment|x-pantheon-styx-hostname|wpe-backend|hubspot|192fc2e7e50945beb8231a492d6a8024|x-github-request|alproxy|netlify|x-lw-cache|squarespace|x-wix-request-id|x-shopify-stage|x-now-cache|flywheel|weebly)')
+      '(seravo|x-kinsta-cache|automattic.com/jobs|x-ah-environment|x-pantheon-styx-hostname|wpe-backend|hubspot|192fc2e7e50945beb8231a492d6a8024|x-github-request|alproxy|netlify|x-lw-cache|squarespace|x-wix-request-id|x-shopify-stage|x-now-cache|flywheel|weebly|dps/)')
```
Comment (rviscomi) [inline ttfb.sql]: I just realized the SQL in this query is duplicated for 07 and 08. Would you mind helping fix that by deleting everything before the second `#standardSQL` line? 😁

Comment (asilvas) [reply ttfb.sql]: Was wondering about that hah

**Comment by rviscomi** [comment]
This change is live!
image

Seeing it on the leaderboard makes the high % of fast TTFB really jump out. Certainly makes GoDaddy look good 😁. Are these static websites similar to GitHub pages or Netlify websites? Would love to learn more about what it is that makes these websites so fast.

**Comment by igrigorik** [comment]
^^ ditto, that is pretty impressive!

**Comment by asilvas** [comment]
Appreciate the interest!

While much of the content is static, we have a fair amount of dynamic content as well. "DPS" is a custom purpose-built Node.js web server, distributed and replicated across the globe. There's a bunch of other fun technology involved, but on the topic of TTFB those are the most relevant.

We also published a white paper a couple years back on this topic: https://www.godaddy.com/garage/site-speed-small-business-website-white-paper/

cheers,

**Pull request merged**